File: BookStoreApi/BookStoreApi/Dtos/ImageDto.cs
namespace BookStoreApi.Dtos;

/// <summary>
/// Dto картинки
/// </summary>
public class ImageDto
{
    /// <summary>
    /// Гуид картинки
    /// </summary>
    public Guid ImageGuid { get; set; }

    /// <summary>
    /// Байты картинки
    /// </summary>
    public IEnumerable<byte> ImageBytes { get; set; }
}

File: frontend/src/app/dtos/image.dto.ts
/** Dto для картинок. */
export class ImageDto {
    /** Гуид картинки */
    public guid: string;

    /** Base64 строка картинки. */
    public base64String: string;

    constructor(guid: string, base64String: string) {
        this.guid = guid;
        this.base64String = base64String;
    }
}
